_includes/tag_list.html

```diff
@@ -1,4 +1,12 @@
 <div id="sidebar">
+  {%- if page_title contains "Upgrade" -%}
```

Comment:
@ercohen14 where did you get this variable name? I don't see it in the official Jekyll documentation: https://jekyllrb.com/docs/variables/

I do see `page.title` which might be the problem.

Reply:
This works fine, it applies the ad to only certain blog posts! I just meant to say that Mauro said it would be "better" to use categories instead of page_title, and that is what I couldn't make work.

Reply:
Page_title came from Mauro's comment above